client/jetpack-cloud/sections/partner-portal/payment-methods/stored-credit-card-method.js

```diff
@@ -21,13 +21,16 @@ export function createStoredCreditCardMethod( {
 			<CreditCardFields stripe={ stripe } stripeConfiguration={ stripeConfiguration } />
 		),
 		submitButton: (
-			<CreditCardPayButton
+			<CreditCardSubmitButton
 				store={ store }
 				stripe={ stripe }
 				stripeConfiguration={ stripeConfiguration }
 				activeButtonText={ activePayButtonText }
 			/>
 		),
+		// these are not shown in the Partner Portal, but are required for CheckoutProvider
```

Comment:
I think we typically start comments with a capital letter 🔠 
`// these are not shown in the Partner Portal, but are required for CheckoutProvider`
`// These are not shown in the Partner Portal, but are required for CheckoutProvider`